kitsune/tidings/events.py

```diff
@@ -109,7 +105,24 @@ class Event(object):
     filters = set()
 
     def fire(self, exclude=None, delay=True):
-        """Notify everyone watching the event.
+        """
+        Notify everyone watching the event, either synchronously or asynchronously.
+        """
+        if delay:
+            event_info = self.serialize()
+            if exclude:
+                if not isinstance(exclude, Sequence):
```

Comment:
I do feel that we are adding unnecessary  complexity here. Let's change the calling of this function to always pass a list of ids (optionally) and we can only do:

```python
if not exclude:
    exclude = []
```

In the case where we are dealing with querysets let's use the `values_list` . By doing this we can omit all this check since we will be always passing a list. We can also add that  in the docstring of the fuction